students/kulesza_monika/lesson_01_02/Lesson.py

```diff
@@ -0,0 +1,139 @@
+def sum_of_three_numbers():
+    a = int(input())
+    b = int(input())
+    c = int(input())
```

Comment:
Nice, but try to use different variable names. Nobody know what 'c' means. Change it to 'first_number', 'second_number' etc